scripts/seed-database/container.sh

```diff
@@ -1,26 +1,29 @@
 #!/usr/bin/env bash
-# This is the script that runs inside the container
-# Usage: container.sh <local_pg_url> <remote_pg_url>
+set -eo pipefail
+cd $(dirname ${0})
 
-# cd to this script's directory
-cd "$(dirname "$0")" || exit
+# set db connection strings
+LOCAL_PG=${1}
+REMOTE_PG=${2}
 
-set -ex
+echo downloading public data from production
 
-LOCAL_PG="$1"
-REMOTE_PG="$2"
+# set-up tmp dir for remote data
+mkdir -p /tmp
 
-# fetch users
-psql --command="\\COPY (SELECT * FROM users) TO '/tmp/users.csv' (FORMAT CSV, DELIMITER ';')" "${REMOTE_PG}"
+tables=(flows users teams flow_document_templates)
 
-# fetch teams
-psql --command="\\COPY (SELECT id, name, slug, theme, settings, notify_personalisation FROM teams) TO '/tmp/teams.csv' (FORMAT CSV, DELIMITER ';')" "${REMOTE_PG}"
+# run copy commands on remote  db
+for table in "${tables[@]}"; do
+  target=/tmp/${table}.csv
+  cmd="\\copy (SELECT * FROM ${table}) TO ${target} WITH (FORMAT csv, DELIMITER ';');"
+  psql --quiet ${REMOTE_PG} --command="${cmd}"
+  echo ${table} downloaded
+done
 
-# fetch flows
-psql --command="\\COPY (SELECT * FROM flows) TO '/tmp/flows.csv' (FORMAT CSV, DELIMITER ';')" "${REMOTE_PG}"
+echo beginning write transaction
+psql --quiet ${LOCAL_PG} -f write.sql
 
-# fetch published_flows (the last two)
-psql --command="\\COPY (SELECT id, data, flow_id, summary, publisher_id FROM (SELECT id, data, flow_id, summary, publisher_id, ROW_NUMBER() OVER (PARTITION BY flow_id ORDER BY created_at DESC) as row_num FROM published_flows) as subquery WHERE row_num <= 2) TO '/tmp/published_flows.csv' (FORMAT CSV);" "${REMOTE_PG}"
```

Comment:
Published flows take a long time to download and I'm not certain it's particularly valuable.

The practice I've been following is to ensure a flow is published before testing. If we continue to do this, syncing published flows should be something we can avoid. Is that right?

Reply:
My understanding of why we want to pull down the most recent published flow per flow is that we'd like to be able to share stable `/preview` flow links to testers (eg user research team) who do not have editor logins. If these flows aren't published, then key features like save & return can behave unexpectedly.

Reply:
Thanks. I've added the explicit option to include this table and that will still be part of the staging sync workflow.